reference/enterprise-api-ref/file-changes.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: File Changes API
+published: true
+tags: [reference, enterprise, API, reporting, file changes]
+---
+
+
+## File changes statistics
+
+**URI:** https://hub.cfengine.com/api/file-changes/statistics?fromTime=:fromTime&toTime=:toTime
+
+**Method:** GET
+
+Get file changes statistics by period.
+
+* **fromTime** *(timestamp)*
+    Include changes performed within interval.
+* **toTime** *(timestamp)*
+    Include changes performed within interval. 
+
+**Example request (curl):**
+
+```
+curl -k --user <username>:<password> \
+  -X GET \
+  "https://hub.cfengine.com/api/file-changes/statistics?fromTime=2019-07-01%2008:56&toTime=2019-07-08%2023:56"
+'
+```
+
+**Example response:**
+
+```
+{
+    "data": {
+        "DIFF": {
+            "20 Sep 2019": 0,
+            "21 Sep 2019": 0,
+            "22 Sep 2019": 0,
+            "23 Sep 2019": 0,
+            "24 Sep 2019": 0,
+            "25 Sep 2019": 0,
+            "26 Sep 2019": 0,
+            "27 Sep 2019": 23
```

Comment:
what do the numbers 0, 23, 33 mean as values of the various dates in the assoc array? Would be good to explain the structure of the response.

Reply:
added